### ISSUE TITLE
Feature request: fetch places by slug

### ISSUE THREAD

**Issue by simonw** [inaturalist/iNaturalistAPI#121]
I noticed some places have a neat human-readable `slug` property, for example New York: https://api.inaturalist.org/v1/places/48

      ...
      "id": 48,
      "display_name": "New York, US",
      "slug": "new-york-us"
      ...

It would be neat if it was possible to use this slug in calls to the `/v1/places/` API to retrieve the correct records, for example being able to hit this URL to get back New York:

https://api.inaturalist.org/v1/places/new-york-us

Since slugs don't appear to contain commas, this could even allow for multi-get - so one could fetch New York and New Jersey either like this:

https://api.inaturalist.org/v1/places/48,51

Or like this:

https://api.inaturalist.org/v1/places/new-york-us,new-jersey

**Comment by simonw** [comment]
The reason I'd like to have this: I'm building an app against the iNaturalist API which has its own place pages. Right now I have to use the integer IDs in my URLs - if I could fetch places by their slug I could use the slugs in my own URLs which would be more aesthetically pleasing.

**Comment by simonw** [comment]
Here's what I'm talking about https://www.owlsnearme.com/?place=48

**Comment by pleary** [comment]
Nice job on the owls app!

This feature has been added and deployed, so people can now look up places (and projects) by their `slug` strings. Multiple values are allowed just the same as IDs. Looks like it's working on your end now too https://www.owlsnearme.com/?place=new-york-us

I see you're using React for your app. We use React for most of our new web UIs (like the observations and taxa show pages), and we created an `inaturalistjs` codebase to handle most communication between the front-end and the API https://github.com/inaturalist/inaturalistjs (we also use it on the server-side with Node.js). It would be interesting to know if it was at all helpful for you as I'm not aware of other developers using it yet. Documentation is pretty lacking, and it supports a lot more methods than are currently listed there. But if that codebase was useful to other developers, I'd be motivated to clean up the README
